### ISSUE TITLE
UI: Highlight files on which some operation is being executed in the background (e.g. delete)

### ISSUE THREAD

**Issue by Najmahan** [aces/cbrain#297]
When a user chooses a number of files and performs a file operation, such as delete, or move, then it would be useful to highlight those files in the file view. The reason is to provide visual cue during the delay between selecting and operation and completion of it.

**Comment by prioux** [comment]
This will be the responsibility of the janitor framework. Right now there's no way to do this properly, since background tasks are, as expected, completely in the background.

**Comment by prioux** [comment]
I'll add a note to the Janitor issue.

**Comment by Najmahan** [comment]
Related, is it possible to have a "janitor" tab maybe next to task to indicate what is happening in the background? For example, I tried to "compress" some civet outputs, 'hoping' that it will create a .tar file. But nothing has happened yet. So I assumed that it was still happening in the background.

(I also tried compressing just one mnc file, I saw a note that "compress finished", but I didn't see any compressed files (gz, zip, tar?, where?)
